Nim In Action SFML test fails on OpenBSD

The Nim In Action SFML test `tests/niminaction/Chapter8/sfml/sfml_test.nim` fails on OpenBSD with the following error:

```
$ nim c -r tests/niminaction/Chapter8/sfml/sfml_test.nim
Hint: used config file '/home/euant/Nim/config/nim.cfg' [Conf]
Hint: used config file '/home/euant/Nim/config/config.nims' [Conf]
Hint: used config file '/home/euant/Nim/tests/config.nims' [Conf]
Hint: system [Processing]
Hint: widestrs [Processing]
Hint: io [Processing]
Hint: sfml_test [Processing]
Hint: sfml [Processing]
Hint: os [Processing]
Hint: strutils [Processing]
Hint: parseutils [Processing]
Hint: math [Processing]
Hint: bitops [Processing]
Hint: macros [Processing]
Hint: algorithm [Processing]
Hint: unicode [Processing]
Hint: pathnorm [Processing]
Hint: osseps [Processing]
Hint: posix [Processing]
Hint: times [Processing]
Hint: options [Processing]
Hint: typetraits [Processing]
Hint: clang -c  -w   -I/home/euant/Nim/lib -I/home/euant/Nim/tests/niminaction/Chapter8/sfml -o /home/euant/.cache/nim/sfml_test_d/stdlib_system.nim.c.o /home/euant/.cache/nim/sfml_test_d/stdlib_system.nim.c [Exec]
Hint: clang -c  -w   -I/home/euant/Nim/lib -I/home/euant/Nim/tests/niminaction/Chapter8/sfml -o /home/euant/.cache/nim/sfml_test_d/stdlib_posix.nim.c.o /home/euant/.cache/nim/sfml_test_d/stdlib_posix.nim.c [Exec]
Hint: clang -c  -w   -I/home/euant/Nim/lib -I/home/euant/Nim/tests/niminaction/Chapter8/sfml -o /home/euant/.cache/nim/sfml_test_d/stdlib_times.nim.c.o /home/euant/.cache/nim/sfml_test_d/stdlib_times.nim.c [Exec]
Hint: clang -c  -w   -I/home/euant/Nim/lib -I/home/euant/Nim/tests/niminaction/Chapter8/sfml -o /home/euant/.cache/nim/sfml_test_d/stdlib_os.nim.c.o /home/euant/.cache/nim/sfml_test_d/stdlib_os.nim.c [Exec]
Hint: clang -c  -w   -I/home/euant/Nim/lib -I/home/euant/Nim/tests/niminaction/Chapter8/sfml -o /home/euant/.cache/nim/sfml_test_d/@msfml_test.nim.c.o /home/euant/.cache/nim/sfml_test_d/@msfml_test.nim.c [Exec]
/home/euant/.cache/nim/sfml_test_d/@msfml_test.nim.c:10:10: fatal error: 'SFML/Graphics.hpp' file not found
#include <SFML/Graphics.hpp>
         ^~~~~~~~~~~~~~~~~~~
1 error generated.
Error: execution of an external program failed: 'clang -c  -w   -I/home/euant/Nim/lib -I/home/euant/Nim/tests/niminaction/Chapter8/sfml -o /home/euant/.cache/nim/sfml_test_d/@msfml_test.nim.c.o /home/euant/.cache/nim/sfml_test_d/@msfml_test.nim.c'
```

SFML is installed with `doas pkg_add sfml`:

```
-rw-r--r--  1 root  bin  186952 Oct 11  2019 /usr/local/lib/libsfml-audio.so.1.0
-rw-r--r--  1 root  bin  474184 Oct 11  2019 /usr/local/lib/libsfml-graphics.so.1.0
-rw-r--r--  1 root  bin  188400 Oct 11  2019 /usr/local/lib/libsfml-network.so.1.0
-rw-r--r--  1 root  bin  107704 Oct 11  2019 /usr/local/lib/libsfml-system.so.1.0
-rw-r--r--  1 root  bin  209368 Oct 11  2019 /usr/local/lib/libsfml-window.so.1.0
```

--- a/config/nim.cfg
+++ b/config/nim.cfg
@@ -205,6 +205,9 @@ clang.objc.options.linker = "-lobjc -lgnustep-base"
   llvm_gcc.cpp.options.linker = "-Wl,-rpath=.:/usr/local/lib:/usr/pkg/lib:/usr/X11R6/lib"
   clang.options.linker = "-Wl,-rpath=.:/usr/local/lib:/usr/pkg/lib:/usr/X11R6/lib"
   clang.cpp.options.linker = "-Wl,-rpath=.:/usr/local/lib:/usr/pkg/lib:/usr/X11R6/lib"
+
+  cincludes: "/usr/local/include"
+  clibdir: "/usr/local/lib"
 @end
 
 # Configuration for the VxWorks

--- a/tests/niminaction/Chapter8/sfml/sfml_test.nim
+++ b/tests/niminaction/Chapter8/sfml/sfml_test.nim
@@ -1,9 +1,6 @@
 discard """
 action: compile
 disabled: "windows"
-disabled: "freebsd"
-disabled: "openbsd"
-disabled: "netbsd"
 """
 
 import sfml, os

